docs/hub/accounts.md

```diff
@@ -1,35 +1,88 @@
+Open the contents of the downloaded archive and run the `install` script. This should install the `hub` tool on your computer. You can verify with `hub --help
 
-#### Mac Installation
+![Run the hub install script.](../images/install/install.png)
 
 On MacOS you will need to confirm that it is okay to run the `hub` binary before it runs successfully. You will see this warning:
 
 ![Select 'Cancel'](../images/install/install_warn.png)
 
-![Go to 'System Settings' => 'General' where you will be able to click, 'Allow Anyway'](../images/install/install_allow.png)
+Select 'Cancel.' 
 
-![Run `hub --help` again and this time select, 'Open` when prompted.](../images/install/install_confirm.png)
+Go to 'System Settings' => 'General' where you will be able to click, 'Allow Anyway.'
+
+![System Settings](../images/install/install_allow.png)
+
+Run `hub --help` again and this time select, 'Open` when prompted.
+
+![Select 'Open` when prompted.](../images/install/install_confirm.png)
 
 You will now be able to continue using the CLI without issue. When you update the binary, you may need to repeat the above steps. We will add developer signing to our MacOS binaries soon.
 
 ![Successfully running `hub --help` to print out the command docs.](../images/hub-cli/hub_help.png)
 
+## Windows Installation
+
+The Windows Hub CLI binary comes with a `hub.exe` that runs `hub` commands. 
+
+You can use [Powershell](https://docs.microsoft.com/en-us/powershell/) to work with the hub CLI by running the executable directly in your terminal like below.
+
+![Running hub --help in Powershell.](../images/install/windows_hub_help.png)
+
+This method requires you to work in the same directory as `hub.exe`.
+
+Another option is to add `hub.exe` to the PATH environment variable to access the `hub` command globally.
+
+### Adding hub.exe to the Path Environment Variable
+
+Place the hub folder in an accessible location like the `C:\` directory.
+
+![Location of hub.exe](../images/install/windows_hub_directory.png)
+
+Open your environment variables option by typing in 'Environment variables' in the Windows start menu and select the first option.
+
+![Search for the environment variables.](../images/install/windows_hub_env1.png)
+
+Select 'Environment Variables' at the bottom.
+
+![Click Environment Variables](../images/install/windows_hub_env2.png)
+
+Select the 'Path' option from User variables if you want to have the hub CLI available only to the current user, or from System variables if you want the hub CLI available for the whole system.
+
+After 'Path' is selected, click 'Edit.'
+
+![Edit your Path.](../images/install/windows_hub_variables.png)
+
+In the 'Edit environment variables' window, select 'New' and enter the location of your hub directory.
+
+![Add hub to environment variables.](../images/install/windows_hub_edit.png)
```

Comment:
maybe it would be worthwhile to create a different directory name in the process. e.g. `textile/` and move the `hub.exe` into it (as opposed to using the folder name created when unzipping). the reason is, that when you update it may conflict with future versions if a windows user does this twice with different directories (based on the version downloaded).

Reply:
Ok sure, will update that ASAP!